tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -1009,3 +1009,26 @@ def test_switch_default_env(self, cluster):
         # set it back
         cluster.default_env = test_env
         cluster.delete(new_env.name)
+
+    @pytest.mark.level("local")
+    @pytest.mark.clustertest
+    # @pytest.mark.skip(
+    #     "Stopping and restarting the server mid-test causes some errors, need to fix"
+    # )
+    def test_enable_and_disable_observability(self, cluster):
```

Comment:
i feel like the more thorough test would be going thru the different ways observability can be set (ex: in the config.yaml, in the restart command, or initially in the constructor), and making sure we extract the right value. 

if the value is set in the cluster servlet properly not sure we need to be stopping and restarting the server anyways